avocado/plugins/archive.py

```diff
@@ -0,0 +1,47 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2016
+# Authors: Cleber Rosa <crosa@redhat.com>
+
+"""Result Archive Plugin"""
+
+from avocado.core.plugin_interfaces import CLI, Result
+from avocado.utils import archive
+
+
+class Archive(Result):
+
+    name = 'zip_archive'
```

Comment:
Yeah, because I cheated.  :smile: 

Now, to better explain it: it seems to make sense to have archive run last, to get all the job results dir content.  But, I felt like adding entries to the configuration file by default was too much.  Since this archive zips the content, the *easy* fix was to name it **z**ip_archive.